### DISCUSSION TITLE
Resource Packs

### DISCUSSION THREAD

**Discussion by bschurman1** [bridge-core/bridge.#13] [Q&A] [answered]
I saw in DanRobzProbz youtube video that he had a resource pack button on the left hand side. However, I don’t have that button. Is that intentional? I really don’t know what to do then when it comes to making completely new entities

**Comment by CapCash303** [comment, marked as answer]
That is because he is using a pre-release of the program, this can be downloaded by clicking on **<> Code**, scrolling down to **Download** and click **Latest version!**. Than click **Releases** to see all the current releases. Be aware these pre-releases may have bugs that is why there not fully released to the main stream yet.

![Capture](https://user-images.githubusercontent.com/52662300/60811103-73d91c80-a143-11e9-9ad5-0d0c85979bc4.PNG)

![Capture](https://user-images.githubusercontent.com/52662300/60810967-1f35a180-a143-11e9-9537-6b53c39f0f99.PNG)

**Comment by solvedDev** [comment]
Yes, @CapCash303 gave a pretty good summary of how to get RP support with the bridge. pre-releases. Closing the issue now :)
